insights/parsers/__init__.py

```diff
@@ -277,3 +277,17 @@ def parse_fixed_table(table_lines,
         table_data.append(col_data)
 
     return table_data
+
+
+def keyword_search(rows, **kwargs):
+    """
+    Takes a list of dictionaries and finds all the dictionaries where the
+    keys and values match those found in the keyword arguments.
+    """
+    results = []
+    if not kwargs:
+        return results
+    for line, row in enumerate(rows):
```

Comment:
That's probably been carried over from my more convoluted logic - good catch, @xiangce, will remove that.